tests/schema_registry/test_proto.py

```diff
@@ -83,3 +111,27 @@ def test_index_encoder(msg_idx, zigzag, expected_hex):
     buf.seek(0)
     decoded_msg_idx = ProtobufDeserializer._decode_index(buf, zigzag=zigzag)
     assert decoded_msg_idx == msg_idx
+
+
+@pytest.mark.parametrize("pb2, data", TEST_DATA)
+def test_round_trip(mock_schema_registry, pb2, data):
```

Comment:
Please write a better name for the test which suggests what we are trying to test. Something in the line of serializing, deserializing and then again serializing. Better to have verbose test name.